src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import cn from 'classnames';
+import { PeopleTableRow } from '../PeopleTableRow';
+import { SearchLink } from '../SearchLink';
+import { Person } from '../../types/Person';
+import { TableRowLabel } from '../../types/TableRowLabel';
+import { capitalize } from '../../utils/capitalize';
+
+type Props = {
```

Comment:
Similar names are okay for a small project, but on large projects we will have many interfaces and types, so it is better to give more detailed names, for example `PeopleTableProps`.
This applies to all code.